openfl/utilities/fed_timer.py

```diff
@@ -0,0 +1,191 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Components Timeout Configuration Module"""
+
+import asyncio
+import logging
+import os
+import time
+
+from contextlib import contextmanager
+from functools import wraps
+from threading import Thread
+
+logger = logging.getLogger(__name__)
+
+
+class CustomThread(Thread):
+    '''
+    The CustomThread object implements `threading.Thread` class.
+    Allows extensibility and stores the returned result from threaded execution.
+
+    Attributes:
+    target (function): decorated function
+    name (str): Name of the decorated function
+    *args (tuple): Arguments passed as a parameter to decorated function.
+    **kwargs (dict): Keyword arguments passed as a parameter to decorated function.
+
+    '''
+    def __init__(self, group=None, target=None, name=None, args=(), kwargs={}):
+        Thread.__init__(self, group, target, name, args, kwargs)
```

Comment:
Why not `super().__init__()`?

Reply:
I think its pretty much the same thing.
using super() would benefit only during multiple inheritance situation like `class C(A, B)`.